src/daft-plan/src/physical_ops/scan.rs

```diff
@@ -35,6 +35,21 @@ impl TabularScan {
         res.push(format!("Num Scan Tasks = {num_scan_tasks}",));
         res.push(format!("Estimated Scan Bytes = {total_bytes}",));
 
+        #[cfg(feature = "python")]
+        if let FileFormatConfig::Database(..) = self.scan_tasks[0].file_format_config.as_ref() {
+            let sql_queries = self
+                .scan_tasks
+                .iter()
+                .map(|st| {
+                    st.file_format_config
+                        .multiline_display()
+                        .pop()
+                        .expect("Each scan task should have a single SQL query")
+                })
+                .join(", ");
+            res.push(format!("SQL Queries = [{}]", sql_queries));
+        }
+
```

Comment:
Trying to think of a better place to display the sql queries instead of via this branch in the tabular scan display. Would have been nice to do it in the optimized logical plan instead but the sql queries are only generated upon creation of physical plan. Lmk if you have any thoughts!

Reply:
We could display it in the optimized logical plan if we generate the sql queries on-the-fly when display of a LogicalPlan is invoked perhaps?